test/XRP/issued-currency-client.test.ts

```diff
@@ -0,0 +1,73 @@
+import { assert } from 'chai'
+
+import { Utils, WalletFactory, XrplNetwork } from 'xpring-common-js'
+import IssuedCurrencyClient from '../../src/XRP/issued-currency-client'
+import {
+  FakeXRPNetworkClient,
+  FakeXRPNetworkClientResponses,
+} from './fakes/fake-xrp-network-client'
+import 'mocha'
+
+const fakeSucceedingNetworkClient = new FakeXRPNetworkClient()
+// const fakeErroringNetworkClient = new FakeXRPNetworkClient(
+//   FakeXRPNetworkClientResponses.defaultErrorResponses,
+// )
+const walletFactory = new WalletFactory(XrplNetwork.Test)
+
+describe('Default XRP Client', function (): void {
+  it('Dummy description', function (): void {
+    // GIVEN an IssuedCurrencyClient.
+    const issuedCurrencyClient = new IssuedCurrencyClient(
+      fakeSucceedingNetworkClient,
+      XrplNetwork.Test,
+    )
+
+    // WHEN the compiler is told to shut up THEN everything is fine.
+    issuedCurrencyClient.shutUpCompiler()
+  })
+  it('Enable Deposit Auth - successful response', async function (): Promise<
```

Comment:
```suggestion
  it('requireAuthorizedTrustlines - successful response', async function (): Promise<
```

Reply:
Not sure if we want to call this "`Require Authorized Trust Lines`" the way Keefer styled the `DefaultXrpClient` tests... or reflect the method name directly like I've done here... ?  the method name directly always made more sense to me.